homeassistant/components/starlink/binary_sensor.py

```diff
@@ -60,64 +60,63 @@ def is_on(self) -> bool | None:
 BINARY_SENSORS = [
     StarlinkBinarySensorEntityDescription(
         key="update",
-        name="Update available",
```

Comment:
Since 2023.7, to reduce the amount of translations they introduced a way to use the device classes. So removing name and translation key would make it fall back to the default translations